app/models/campaign.rb

```diff
@@ -0,0 +1,63 @@
+class Campaign < ApplicationRecord
```

Comment:
Why the validations are not being handled at the backend too just like frontend ? like start_time should be less than end_time and others ?